src/uci.cpp

```diff
@@ -40,12 +40,12 @@ extern vector<string> setup_bench(const Position&, istream&);
 
 namespace {
 
-  // FEN string of the initial position, normal chess
+  // A FEN string for the initial position in standard chess.
```

Comment:
the leading indefinite article here is incorrect. probably I would say "the FEN string" here, tho a plain "FEN string" as is is also fine